src/features/todo/Todo.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react'
+import { useDispatch, useSelector } from 'react-redux'
+import { createTask, isCompleted, editTask, deleteTask } from './todoSlice'
+import { useState } from 'react'
```

Comment:
No need to have to import statements from the same package/file, you can merge default imports and named imports like this:

```jsx
import React, { useState } from 'react';
```